Packs/ContentTesting/Scripts/UnitTestPlaybookAnalyzer/UnitTestPlaybookAnalyzer.py

```diff
@@ -1,204 +1,129 @@
+                tasks.append(BuildTask(t))
 
     return tasks
 
 
-def TaskStats(task: list[Task], taskstat: dict[str, TaskStat]) -> dict[str, TaskStat]:
+def TaskStats(task: list, taskstat: dict) -> dict:
     for t in task:
-        tid = t['tid']
-        dur = t['duration']
-        if tid not in taskstat:
-            taskstat[tid] = {'tid': tid, 'name': t['name'], 'mindur': 0, 'maxdur': 0, 'avgdur': 0, 'totdur': 0,
-                             'count': 0, 'completed': 0, 'started': 0, 'notexecuted': 0, 'error': 0, 'waiting': 0}  # type: ignore
+        taskid = t['tid']
+        dur = int(t['duration'])
+        if taskid not in taskstat:
+            taskstat[taskid] = {
+                'tid': taskid,
+                'name': t['name'],
+                'mindur': 1000000,
+                'maxdur': 0,
+                'avgdur': 0,
+                'totdur': 0,
+                'count': 0,
+                'completed': 0,
+                'started': 0,
+                'notexecuted': 0,
+                'error': 0,
+                'waiting': 0
+            }
         if t['state'] == "Completed":
-            if dur > taskstat[tid]['maxdur']:  # type: ignore
-                taskstat[tid]['maxdur'] = dur  # type: ignore
-            if taskstat[tid]['mindur'] is not None:  # type: ignore
-                if dur < taskstat[tid]['mindur']:  # type: ignore
-                    taskstat[tid]['mindur'] = dur  # type: ignore
-            else:
-                taskstat[tid]['mindur'] = dur  # type: ignore
-            taskstat[tid]['totdur'] += dur  # type: ignore
-            taskstat[tid]['completed'] += 1  # type: ignore
+            if dur > taskstat[taskid]['maxdur']:
+                taskstat[taskid]['maxdur'] = dur
+            if dur < taskstat[taskid]['mindur']:
+                taskstat[taskid]['mindur'] = dur
+            taskstat[taskid]['totdur'] += dur
+            taskstat[taskid]['completed'] += 1
         elif t['state'] == "Error":
-            taskstat[tid]['error'] += 1  # type: ignore
+            taskstat[taskid]['error'] += 1
         elif t['state'] == "Waiting":
-            taskstat[tid]['waiting'] += 1  # type: ignore
+            taskstat[taskid]['waiting'] += 1
         else:
-            taskstat[tid]['started'] += t['started']  # type: ignore
-            taskstat[tid]['notexecuted'] += t['notexecuted']  # type: ignore
-        taskstat[tid]['count'] += 1  # type: ignore
+            taskstat[taskid]['started'] += t['started']
+            taskstat[taskid]['notexecuted'] += t['notexecuted']
+        taskstat[taskid]['count'] += 1
 
     for key, ts in taskstat.items():
         ts['avgdur'] = int(ts['totdur'] / ts['count'])
 
     return taskstat
 
 
-def GetTaskStats(playbookname: str, occurred: str) -> tuple[dict[str, TaskStat], int]:
-    argument = {'query': f'playbook:"{playbookname}" occurred:>="{occurred}"', 'size': 1000}
+def GetTaskStats(playbookname, subplaybookname, firstday, lastday, maxinc):
```

Comment:
The suggestion in order to keep 'occurred':

```suggestion
def GetTaskStats(playbookname, subplaybookname, firstday, lastday, maxinc, occurred):
```